def ValidatorException(Exception):
    pass

def RepoException(Excpetion):
    pass